package.json

```diff
@@ -10,15 +9,20 @@
     "datatables.net-rowreorder-bs4": "^1.2.8",
     "jquery": "^3.5.0",
```

Comment:
What do we still use jquery for? Is there legacy javascript code that is using it?